### PR TITLE
Add changelog for 1.0

### PR DESCRIPTION
**Pull request by kaxil**
Adds changelog for 1.0

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,47 +1,79 @@
...
+       case of BigQuery and Snowflake [COPY INTO](https://docs.snowflake.com/en/sql-reference/sql/copy-into-table.html)
+       using [external stage](https://docs.snowflake.com/en/sql-reference/sql/create-stage.html) in the case of Snowflake.
+    - `Pandas`: This is how datasets were previously loaded. To enable this mode, use the argument
+      `use_native_support=False` in `aql.load_file`.
+    - `Hybrid`: This attempts to use the native strategy to load a file to the database and if native strategy(i)
+       fails , fallback to Pandas (ii) with relevant log warnings.
+* Allow users to specify the table schema (column types) in which a file is being loaded by using `table.columns`.
+  If this table attribute is not set, the Astro SDK still tries to infer the schema by using Pandas
+  (which is previous behaviour).[#532](https://github.com/astronomer/astro-sdk/issues/532)
+* Implement fallback mechanism in case native support fails to default option with log warning for problem with native
```
Comment (utkarsharma2) [inline docs/CHANGELOG.md:29]: @kaxil we already talked about this in the `Hybrid` section, do we need to add this again?

Comment (kaxil) [reply docs/CHANGELOG.md:29]: Good point, we should remove it

**Review by utkarsharma2 — APPROVED** [review]

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,47 +1,77 @@
...
 * Get configurations via Airflow Configuration manager. [#503](https://github.com/astronomer/astro-sdk/issues/503)
-* Add CI job to check for dead links [#526](https://github.com/astronomer/astro-sdk/pull/526)
+* Change catching `ValueError` and `AttributeError` to `DatabaseCustomError` [#595](https://github.com/astronomer/astro-sdk/pull/595)
+* Unpin pandas upperbound dependency [#620](https://github.com/astronomer/astro-sdk/pull/620)
+* Remove markupsafe from dependencies [#623](https://github.com/astronomer/astro-sdk/pull/623)
+* Added `extend_existing` to Sqla Table object [#626](https://github.com/astronomer/astro-sdk/pull/626)
+* Move config to store DF in XCom to settings file [#537](https://github.com/astronomer/astro-sdk/pull/537)
+* Make the operator names consistent [#634](https://github.com/astronomer/astro-sdk/pull/634)
+* Use `exc_info` for exception logging [#643](https://github.com/astronomer/astro-sdk/pull/643)
+* Update query for getting bigquery table schema  [#661](https://github.com/astronomer/astro-sdk/pull/661)
```
Comment (utkarsharma2) [inline docs/CHANGELOG.md:67]:
```suggestion
```

Comment (utkarsharma2) [reply docs/CHANGELOG.md:67]: @kaxil this was just for the test case and not our issue with our codebase. I think we can remove this too.

Comment (kaxil) [reply docs/CHANGELOG.md:67]: Oh yes, you are right

**Pull request merged**